README.md

```diff
@@ -70,17 +70,29 @@ Beta features:
   - [ ] Use network libraries in the Outline Client
   - [ ] Add extensive testing
 
-- Serverless transport libraries
-  - [ ] Encrypted DNS
-  - [ ] Packet splitting
-
-- Proxy transport libraries
-  - [ ] HTTP Connect
-  - [ ] SOCKS5
-
-- Add Resources
+- Transport client strategies
+  - Proxyless strategies
+    - [ ] Encrypted DNS
+    - [ ] Packet splitting
+  - Proxy-based strategies
+    - [ ] HTTP Connect
+    - [ ] SOCKS5
+
+- Integration resources
+  - For Mobile apps
```

Comment:
Do we really want to distinguish "mobile app" and "go app"? A "mobile" app can include "go" implementations as well.

Reply:
They are disjoint. No one writes mobile apps in Go. So we need to explain how to integrate Go into mobile apps.